Dockerfile

```diff
@@ -1,15 +1,15 @@
-FROM golang:1.9
+FROM golang:1.9-alpine
 
 COPY cmd/freegeoip/public /var/www
 
 ADD . /go/src/github.com/apilayer/freegeoip
-RUN \
-	cd /go/src/github.com/apilayer/freegeoip/cmd/freegeoip && \
-	go get -d && go install && \
-	apt-get update && apt-get install -y libcap2-bin && \
-	setcap cap_net_bind_service=+ep /go/bin/freegeoip && \
-	apt-get clean && rm -rf /var/lib/apt/lists/* && \
-	useradd -ms /bin/bash freegeoip
+WORKDIR /go/src/github.com/apilayer/freegeoip/cmd/freegeoip
+RUN apk update
+RUN apk add git libcap shadow
+RUN go get -d
```

Comment:
A rebuild produced this error

```
Step 10/13 : RUN useradd -ms /bin/bash freegeoip
 ---> Running in afcee3ec8484
Creating mailbox file: No such file or directory
```
but otherwise image rebuilt and installed OK fully functioning on :80
still outstanding issue on :443 [#3](https://github.com/apilayer/freegeoip/issues/3)

Reply:
Yeah, I looked up that error hoping I could fix it, however after my reserach it seemed like there wasn't a great way to fix it, and it doesn't cause issues. I could create the directory that it's trying to access but I was afraid that would only add to the image size (not sure of that though)?

Reply:
According to this ticket, https://github.com/mhart/alpine-node/issues/48, the correct way to add a user in Apline 3.7 is:
```
RUN \
      addgroup -g 1000 -S freegeoip && \
      adduser -u 1000 -S freegeoip -G freegeoip
```
The base image `golang:1.9-alpine` is currently running 3.6.2, but this still seems to correctly create the home directory:
```
echo $HOME
/home/freegeoip
```

Reply:
followed @dsperling advice, seems the errors are gone @OlagStegan